### PR TITLE
Added signUp (POST /users) swagger doc

### PR DESCRIPTION
**Pull request by andreswolox**
## Summary  [Card](https://trello.com/c/ztLzHKvO)

- Added successfull `signUp (POST /users) 201` response Swagger Doc.
- Added `signUp (POST /users) 422`  possible response errors.
- Added `signUp (POST /users) 503` response error.
- Fixed minor detail in `users repository` error message.

### CODE REVIEW

**Comment by jersonlopez** [comment]
@andreswolox please change the target branch for this pr

File: app/repositories/users.js [hunk 1]
```diff
@@ -27,7 +27,7 @@ class UserRepository {
       return (await this.User.create(user)).toJSON();
     } catch (error) {
       logger.error('Error when was trying to create a new user: ', error.message);
-      throw errors.databaseError('Unknown when was trying to create the user');
+      throw errors.databaseError('Unknown error when was trying to create the user');
```
Comment (jersonlopez) [inline app/repositories/users.js:30]: 👌🏻, my mistake

**Review by jersonlopez — APPROVED** [review]
good job bro! 💯

File: app/repositories/users.js (hunk 1 above)
Comment (andreswolox) [reply app/repositories/users.js:30]: No problem :)

**Pull request merged**